Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -990,7 +987,7 @@ void MainWidget::open_image(Core::File& file)
     auto& editor = create_new_editor(image);
     editor.set_loaded_from_image(m_loader.is_raw_image());
     editor.set_path(file.filename());
-    editor.undo_stack().set_current_unmodified();
```

Comment:
These changes look good in general. I'm wondering, do we still need classes outside of the ImageEditor to touch its undo stack still? i.e. could we make undo_stack() private or remove it?